config/babel.dev.js

```diff
@@ -17,9 +17,10 @@ module.exports = {
   ],
   plugins: [
     require.resolve('babel-plugin-syntax-trailing-function-commas'),
-    require.resolve('babel-plugin-transform-async-to-generator'),
+    require.resolve('babel-plugin-syntax-async-functions'),
     require.resolve('babel-plugin-transform-class-properties'),
     require.resolve('babel-plugin-transform-object-rest-spread'),
+    require.resolve('babel-plugin-transform-regenerator'),
```

Comment:
It is, but in `preset-es2015` it's passed [different options](https://github.com/babel/babel/blob/c1a597ffdc65363dc50136ae63463fb7341a13cc/packages/babel-preset-es2015/index.js#L23), `{ async: false, asyncGenerators: false }`. The default options (used here) are:
```js
{
  asyncGenerators: true,
  generators: true,
  async: true
}
```
(Note that `asyncGenerators` is `true` by default, but since we don't enable the async generator syntax plugin, this doesn't matter.)